internal/controllers/import_controller.go

```diff
@@ -158,7 +158,6 @@ func (r *CAPIImportReconciler) reconcileNormal(ctx context.Context, capiCluster
 	rancherCluster *rancher.Cluster,
 ) (ctrl.Result, error) {
 	log := log.FromContext(ctx)
-	log.Info("Reconciling CAPI Cluster")
```

Comment:
is there reason why it was removed?

Reply:
there is an exactly the log line in `Reconcile()`, this one is a duplicate